Apps/APN/ios/.gitignore

```diff
@@ -0,0 +1,97 @@
+test-report.*
```

Comment:
The contents of this file look good. I suggest that we move all of the contents of this file into `Apps/.gitignore` so all sample apps can use these rules.

Reply:
Moved in [this commit](https://github.com/customerio/customerio-reactnative/pull/136/commits/b38d4031ed106b030e5029cac8a2d07ed25ac044)